index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./styles.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+      integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <title>portfolio</title>
+  </head>
+  <body>
+    <header class="nav">
+      <h3>john doe</h3>
+      <i class="fa-solid fa-bars"></i>
+    </header>
+    <main class="main">
+      <h1>Hey There I`m Nick</h1>
+      <h5>i am a software developer</h5>
+      <p>
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hestiate to contact me.
+      </p>
+      <ul>
+        <li>
+          <i class="fa-brands fa-github"></i>
+        </li>
+        <li>
+          <i class="fa-brands fa-linkedin"></i>
```

Comment:
- [ ] Kindly place this icon in an `a` tag to make it a link that will direct the user to that specific social media.